server/apps/immich/src/modules/immich-auth/strategies/user-auth-strategy.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { AuthService, UserResponseDto, UserService } from '@app/domain';
+import { Strategy } from 'passport-custom';
+import { Request } from 'express';
+
+export const AUTH_COOKIE_STRATEGY = 'auth-cookie';
+
+@Injectable()
+export class UserAuthStrategy extends PassportStrategy(Strategy, AUTH_COOKIE_STRATEGY) {
+  constructor(private userService: UserService, private authService: AuthService) {
+    super();
+  }
+
+  async validate(request: Request): Promise<UserResponseDto> {
```

Comment:
@jrasm91 I think we need to return `AuthUserDto` here to have the same form with `api-key` and `public-share` strategy, corret?

Reply:
Yes, this should return an `AuthUserDto`, I believe it did before as well so we should keep that code/interface.

Reply:
Agreed, must've got changed in the refactor, change that now